src/taoensso/nippy.clj

```diff
@@ -119,6 +119,7 @@
 
    90  [:date     [[:bytes 8]]]
    91  [:uuid     [[:bytes 16]]]
+   92  [:sql-date [[:bytes 8]]]
```

Comment:
heads up that `92` was a random choice of mine without fully understanding the numbering scheme/implications here

Reply:
Thanks for the heads-up. 92 is totally fine, there's no scheme - any random [available](https://github.com/ptaoussanis/nippy/blob/d0eb2d9871fb8733c9f99d0f3b758c4aa305a7e1/src/taoensso/nippy.clj#L243) id is good 👍